### PR TITLE
Add fmt.Sprintf() for logging status conditions

### PR DESCRIPTION
**Pull request by sam-nguyen7**
add in fmt.Sprintf for simplicity and to keep lines shorter. These changes got left out of PR #20.

### CODE REVIEW

File: pkg/controller/subjectpermission/subjectpermission_controller.go [hunk 1]
```diff
@@ -156,7 +157,8 @@ func (r *ReconcileSubjectPermission) Reconcile(request reconcile.Request) (recon
 			clusterRoleNames = append(clusterRoleNames, clusterRoleName)
 
 			// update the condition if creation of a ClusterRoleBinding has failed
-			instance := controllerutil.UpdateCondition(instance, "Unable to create ClusterRoleBinding: "+err.Error(), clusterRoleNames, true, managedv1alpha1.SubjectPermissionFailed)
+			unableToCreateCRBmsg := fmt.Sprintf("Unable to create ClusterRoleBinding: %s", err.Error())
```
Comment (jharrington22) [inline pkg/controller/subjectpermission/subjectpermission_controller.go]: `unableToCreateCrbMsg` to be consistent

File: pkg/controller/subjectpermission/subjectpermission_controller.go [hunk 2]
```diff
@@ -193,7 +195,8 @@ func (r *ReconcileSubjectPermission) Reconcile(request reconcile.Request) (recon
 		permissionsClusterRoleNames = append(permissionsClusterRoleNames, permissionClusterRoleName)
 
 		// update condition
-		updatedSubjectPermission := controllerutil.UpdateCondition(instance, permissionClusterRoleName+" for clusterPermission does not exist", permissionsClusterRoleNames, true, managedv1alpha1.SubjectPermissionFailed)
+		permissionCRNameFailsMsg := fmt.Sprintf("%s for clusterPermission does not exist", permissionClusterRoleName)
```
Comment (jharrington22) [inline pkg/controller/subjectpermission/subjectpermission_controller.go]: `permissionCrNameFailsMsg` here too

File: pkg/controller/subjectpermission/subjectpermission_controller.go (hunk 1 above)
Comment (sam-nguyen7) [reply pkg/controller/subjectpermission/subjectpermission_controller.go]: fixed

File: pkg/controller/subjectpermission/subjectpermission_controller.go (hunk 2 above)
Comment (sam-nguyen7) [reply pkg/controller/subjectpermission/subjectpermission_controller.go]: fixed

**Pull request merged**